communitiesuk/funding-service-design-fund-store
# Pushing fund changes

Adding support for 'mark this section as complete' at the end of each section - new field in the round table.

Goes with the following PRs:
https://github.com/communitiesuk/digital-form-builder/pull/309
https://github.com/communitiesuk/funding-service-design-application-store/pull/228
https://github.com/communitiesuk/funding-service-design-frontend/pull/393


### Change description
_A brief description of the pull request_

- [n/a] README and other documentation has been updated / added (if needed)


### How to test
Update a round to have `markAsCompleteEnabled=true`
Start an application in that round
Complete a section and you should see the 'mark this section as complete' radio fields on the summary
Select 'No'
Back on the tasklist the section should show as 'In PRogress'
Go through the section again and choose 'yes' on the summary page
That section should show as complete in the tasklist


### Screenshots of UI changes (if applicable)
![Screenshot 2023-09-25 at 15 39 01](https://github.com/communitiesuk/funding-service-design-application-store/assets/1729216/f9037857-081f-43cc-b03f-0627fab90bb5)
